contributing_to_docs/doc_guidelines.adoc

```diff
@@ -61,7 +61,8 @@ would be appropriate for a topic titled Managing Authorization Policies.
 a|- Use title case in all topic titles and section headings. See
 http://titlecase.com/ for more information.
 - Try to be as descriptive as possible with the topic title or section headings
-without making them unnecessarily too long.
+without making them unnecessarily too long. Use a verb form in headings (for example,
+Creating, Managing, Using, etc.) whenever possible.
```

Comment:
I'd wager that most readers just quickly scan the overview. I don't think it's a good place to put crucial information.